src/Kdyby/ElasticSearch/Diagnostics/Panel.php

```diff
@@ -108,7 +108,13 @@ public function getPanel()
 				return !is_array($data) ? Json::decode($data, Json::FORCE_ARRAY) : $data;
 
 			} catch (Nette\Utils\JsonException $e) {
-				return $data;
+				try {
+					return array_map(function($row) {
+						return Json::decode($row, Json::FORCE_ARRAY);
+					}, explode("\n", $data));
```

Comment:
Can this be considered safe? What if the json contains `\n`? Would it be always properly escaped?